### DISCUSSION TITLE
Capstone - Bikesharing API DatabaseError

### DISCUSSION THREAD

**Discussion by sultanardian** [teamalgoritma/community#382] [Q&A] [answered]
tidak bisa membaca database, padahal code sudah sesuai dengan video ke 6. flask app and endpoint

Error :
![image](https://user-images.githubusercontent.com/102045793/193456909-fb3acf92-f1f7-451f-9f30-bdc486c46cc7.png)

Code : 
![image](https://user-images.githubusercontent.com/102045793/193456926-46e0363b-3dab-4bce-a962-b25bf636cd64.png)

Database :
![image](https://user-images.githubusercontent.com/102045793/193456976-e6382424-5130-4f10-96af-ecf5ac623874.png)

**Comment by kevbow** [comment, marked as answer]
Halo @sultanardian,

Selalu tutup tableplus ketika menjalankan app.py karena pada kasus ini kita hanya bisa membuka menggunakan 1 koneksi dalam waktu bersamaan, serta coba cek kembali apakah penulisan connection sudah benar atau belum.

Semoga membantu.
